### PR TITLE
US123881 - Convert to GitHub Actions

### PR DESCRIPTION
**Pull request by devpow112**
This removes TravisCI in favour of GitHub Actions. Configs and other formatting were cleaned up as a consequence of clean-up of the eslint config and the addition of the license checker.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+  verify-languages:
+    name: Verify Languages
+    timeout-minutes: 5
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: Brightspace/third-party-actions@actions/checkout
+      - name: Verify Languages
+        run: |
+          status=0
```
Comment (devpow112) [inline .github/workflows/ci.yml:65]: Can this logic be replaced by configuring the `Language Checkr` app?

File: .licensechecker.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+	"acceptedScopes": [
+		"d2l"
+	],
+	"manualOverrides": {
```
Comment (devpow112) [inline .licensechecker.json:5]: I'm looking at removing the need for `polymer-cli` which will get rid of most of these overrides. Want to switch to `es-dev-server`.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (mpharoah-d2l) [reply .github/workflows/ci.yml:65]: Possible? I wrote this earlier to make sure that you didn't forget to update the langterms properly, but there might be something else that will do thsi now

**Review by mpharoah-d2l — APPROVED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (devpow112) [reply .github/workflows/ci.yml:65]: Ok, I'll look into it. Thanks for the info 👍

**Pull request merged**